packages/graphql-yoga/src/utils/yogaDefaultFormatError.ts

```diff
@@ -1,46 +1,69 @@
-import { FormatErrorHandler } from '@envelop/core'
 import { createGraphQLError } from '@graphql-tools/utils'
-import { GraphQLError, GraphQLErrorExtensions } from 'graphql'
+import { isGraphQLError } from '@envelop/core'
+import { GraphQLErrorExtensions } from 'graphql'
 
-export const yogaDefaultFormatError: FormatErrorHandler = (
-  err,
+export const yogaDefaultFormatError = ({
+  error,
   message,
   isDev,
-) => {
-  if (err instanceof GraphQLError) {
-    if (err.originalError) {
-      if (err.originalError.name === 'GraphQLError') {
-        return err
+}: {
+  error: unknown
+  message: string
+  isDev?: boolean
+}) => {
+  const dev = isDev || globalThis.process?.env?.NODE_ENV === 'development'
+
+  if (isGraphQLError(error)) {
+    if (error.originalError) {
+      if (error.originalError.name === 'GraphQLError') {
+        return error
       }
       // Original error should be removed
       const extensions: GraphQLErrorExtensions = {
-        ...err.extensions,
+        // @ts-ignore
+        ...error.extensions,
         http: {
           status: 500,
-          ...err.extensions?.http,
+          // @ts-ignore
+          ...error.extensions?.http,
         },
       }
-      if (isDev) {
+      if (dev) {
         extensions.originalError = {
-          message: err.originalError.message,
-          stack: err.originalError.stack,
+          // @ts-ignore
+          message: error.originalError.message,
+          // @ts-ignore
+          stack: error.originalError.stack,
         }
       }
       return createGraphQLError(message, {
-        nodes: err.nodes,
-        source: err.source,
-        positions: err.positions,
-        path: err.path,
+        // @ts-ignore
+        nodes: error.nodes,
+        // @ts-ignore
+        source: error.source,
+        // @ts-ignore
+        positions: error.positions,
+        // @ts-ignore
+        path: error.path,
```

Comment:
Why so many `@ts-ignore`s? If we absolutely have to, always prefer using `@ts-expect-error` with a description.